crates/ruff_memory_usage/src/lib.rs

```diff
@@ -12,3 +12,9 @@ pub fn heap_size<T: GetSize>(value: &T) -> usize {
         .get_heap_size_with_tracker(&mut *TRACKER.lock().unwrap())
         .0
 }
+
+/// Returns `0`, ignoring the memory usage of the value.
+// TODO: Remove this and add `GetSize` support for `OrderSet`.
```

Comment:
Could we add a simple implementation for now that simply iterates over all values and adds the size of those values. That would be better than just zero (even if not a 100% accurate)